circle.yml

```diff
@@ -201,7 +201,6 @@ commands:
           name: Restore system tests node_modules cache
           keys:
             - v{{ .Environment.CACHE_VERSION }}-{{ checksum "platform_key" }}-system-tests-projects-node-modules-cache-{{ checksum "system_tests_cache_key" }}
-            - v{{ .Environment.CACHE_VERSION }}-{{ checksum "platform_key" }}-system-tests-projects-node-modules-cache-
```

Comment:
this will also remove the cache fallback from the `install` step... but i guess that's the intent, i can see how yarn install from existing node_modules would differ from a fresh install (and be less deterministic)